docs/api-status.md

```diff
@@ -79,4 +79,8 @@ Admin.CreateCephFSExport | v0.16.0 | v0.18.0 |
 Admin.RemoveExport | v0.16.0 | v0.18.0 | 
 Admin.ListDetailedExports | v0.16.0 | v0.18.0 | 
 Admin.ExportInfo | v0.16.0 | v0.18.0 | 
-
+API.ListUsersBuckets | v0.16.0 | v0.18.0 | 
+API.ListUsersBucketsWithStat | v0.16.0 | v0.18.0 | 
+API.CreateKey | v0.16.0 | v0.18.0 | 
+API.RemoveKey | v0.16.0 | v0.18.0 | 
+API.SetIndividualBucketQuota | v0.16.0 | v0.18.0 | 
```

Comment:
New preview APIs are added under wrong(**Package: common/admin/nfs**) section. Running `make api-update` should ensure that these are placed correctly under corresponding package headers.

Reply:
fixed